test/integration/__snapshots__/install.test.js.snap

```diff
@@ -635,49 +635,6 @@ exports[`snowpack install config-custom-path: import-map.json 1`] = `
 }"
 `;
 
-exports[`snowpack install config-extends: allFiles 1`] = `
```

Comment:
I'll be honest, I'm not sure why this snapshot changed. It didn't change when the rollup plugin was inline, but changed when the inline plugin was converted to a function.

Reply:
hmm, that is strange. I'll merge locally to confirm that this isn't reproducible

Reply:
Ah, I see the issue, I get this when I run the tests:

"    + Sourcemap is likely to be incorrect: a plugin (snowpack:rollup-plugin-strip-source-mapping) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help"

From my minimal understanding of how sourcemaps work, I think you're safe to follow Rollup's instructions on what to do if you don't move code around:

```
If the transformation does not move code, you can preserve existing sourcemaps by returning null:

return {
  code: transformedCode,
  map: null
};
```

Since you don't actually remove code and just remove a comment that is almost always at the end of a line, I believe this change does not require a new source map.